resources/dicts/events/leader_den/success/outsider.json

```
@@ -100,5 +100,146 @@
         }
       ]
     }
+  },
+  {
+    "interaction_type": "invite",
+    "event_text": "There's one specific cat, m_c, that keeps regularly upstaging c_n's hunting patrols. However, instead of trying to drive {PRONOUN/m_c/object} out, c_n welcomes m_c with open paws.",
+    "reputation": ["any"],
+    "rep_change": 1,
+    "m_c": {
+      "status": ["loner", "rogue", "kittypet"],
+      "age": ["adolescent", "young adult", "adult", "senior adult"],
+      "skill": [
+        "HUNTER,3",
+        "HUNTER,4"
```

```suggestion
      "skill": [
        "HUNTER,3"
```
Skill tags allow for the tagged skill type of the tagged level *and* any higher levels, so you don't need to tag for both `Hunter,3` and `Hunter,4`.  Just `Hunter,3` will cover both of them.  Go ahead and make this adjustment on all your new hunter events.

---

Gonna be 100% honest, did **_not_** know this LOL. Thank you for telling me